docs/walkthrough/loading-datasets.ipynb

```diff
@@ -35,43 +38,65 @@
     "\n",
     "The dataset loader used is available here: https://github.com/substratusai/dataset-k8s-instructions\n",
     "\n",
-    "Create the Dataset resource to load the dataset into Substratus:"
+    "Create the Dataset resource to load the dataset into Substratus:\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "5c7e849f",
    "metadata": {
+    "tags": [
+     "k apply dataset"
+    ],
     "vscode": {
      "languageId": "plaintext"
     }
    },
    "outputs": [],
    "source": [
-    "! kubectl apply -f https://raw.githubusercontent.com/substratusai/substratus/main/examples/datasets/k8s-instructions.yaml"
+    "!kubectl apply -f https://raw.githubusercontent.com/substratusai/substratus/main/examples/datasets/k8s-instructions.yaml"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "e0aa182b",
    "metadata": {},
    "source": [
-    "Verify the dataset became ready by running:"
+    "Verify the dataset became ready by running:\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "baa9b13d",
    "metadata": {
+    "tags": [],
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!kubectl describe dataset k8s-instructions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a8132c75",
+   "metadata": {
+    "tags": [
+     "k get dataset",
+     "remove_cell"
+    ],
     "vscode": {
      "languageId": "plaintext"
     }
    },
    "outputs": [],
    "source": [
-    "! kubectl describe dataset k8s-instructions"
+    "!kubectl get dataset k8s-instructions -o=json"
```

Comment:
is this hidden and only for tests? I think as an end user the describe format is more user friendly.

Reply:
looks like it is since the markdown doesnt show it